api/types/types.go

```diff
@@ -219,9 +240,10 @@ type ContainerState struct {
 	FinishedAt time.Time
 }
 
+// ContainerJSONBase contains response of Remote API:
 // GET "/containers/{name:.*}/json"
 type ContainerJSONBase struct {
-	Id              string
+	ID              string
```

Comment:
Same as above, as it is *serialized* as JSON, I think it would be better to do something like :

```go
ID string `json:"Id"`
```